content/manuals/dhi/how-to/migrate.md

```diff
@@ -240,3 +240,22 @@ ENTRYPOINT [ "python", "/app/image.py" ]
 
 {{< /tab >}}
 {{< /tabs >}}
+
+### Use Gordon AI
```

Comment:
Should this be "Use Gordon" for consistency?  I'm not sure if we use Gordon AI in the docs or elsewhere in the product.

Reply:
Sure, I was trying to have "AI" in headers to help with findability.